multihost_test/bz_automation/test_automation.py

```diff
@@ -94,3 +94,21 @@ def test_cve_2010_3316(self, multihost, bkp_pam_config, compile_myxauth):
         execute_cmd(multihost, 'rm -vfr /root/.xauth')
         execute_cmd(multihost, f"userdel -rf {TUSER}")
         assert int(execute_cmd(multihost, "cat /tmp/xauthlog | wc -l" ).stdout_text.split()[0]) >= 2
+
+    def test_2082442(self, multihost, bkp_pam_config, create_localusers):
+        """
+        :title: pam_faillock prints "Consecutive login failures
+         for user root account temporarily locked" without even_deny_root
+        :id: c1802552-5151-11ed-b8c2-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2126648
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2082442
+        """
+        execute_cmd(multihost, "> /var/log/secure")
+        for file in ['system-auth', 'password-auth']:
+            execute_cmd(multihost, f"sed -i 's/pam_unix.so nullok/pam_unix.so nullok try_first_pass/g' /etc/pam.d/{file}")
+        ssh1 = SSHClient(multihost.client[0].ip, username="local_anuj", password="password123")
```

Comment:
You should execute this command but for `root` instead of `local_anuj`

Reply:
I just saw that, but I think the reporter made a mistake. The bugzilla also states:
```
When `root` deliberately authenticate with incorrect password, below error message is logged in /var/log/secure although the account is not actually locked.

   "Consecutive login failures for user root account temporarily locked"
```

So can you try with root?

Reply:
currently we dont have his facility to do that , with  local_anuj i can find the desired output in the logs . 

see bellow if we try to do it with root:

[root@client1 ~]# su -
Last login: Wed Oct 26 06:04:19 EDT 2022 on pts/1
[root@client1 ~]# 


It does not ask for any password

Reply:
so it must be a non root user to do this test

Reply:
> [root@client1 ~]# su -

That's because you are already logged in as root. Can you try to log in as another user first and then su as root?

Reply:
yes , thats what i am doing here .

Logged in as local_anuj then try to su -

Reply:
This is what i am getting in the logs:

Consecutive login failures for user root account temporarily locked

Reply:
Understood.